src/components/form/examples/row-layout-schema.ts

```diff
@@ -0,0 +1,107 @@
+export const schema = {
+    description: 'This form has the row layout',
+    type: 'object',
+    properties: {
+        info: {
+            title: 'Preferences',
+            description:
+                'These settings will not affect how others see the data. These are only for you.',
+            type: 'object',
+            lime: {
+                layout: {
+                    type: 'row',
+                },
+            },
+            properties: {
+                language: {
+                    type: 'string',
+                    icon: 'globe',
+                    title: 'Language',
+                    description: 'Select the app language',
+                    default: 'ua',
+                    lime: {
+                        component: 'limel-select',
+                    },
+                    oneOf: [
+                        {
+                            type: 'string',
+                            const: 'sv',
+                            title: 'Swedish',
+                        },
+                        {
+                            type: 'string',
+                            const: 'ua',
+                            title: 'Ukrainian',
+                        },
+                        {
+                            type: 'string',
+                            const: 'en',
+                            title: 'English',
+                        },
+                        {
+                            type: 'string',
+                            const: 'am',
+                            title: 'Amharic',
+                        },
+                        {
+                            type: 'string',
+                            const: 'fa',
+                            title: 'Farsi',
+                        },
+                    ],
+                },
+                date: {
+                    type: 'string',
+                    icon: 'calendar',
+                    title: 'Date format',
+                    default: 'yyyy-mm-dd',
+                    lime: {
+                        component: 'limel-select',
```

Comment:
```suggestion
                        component: {name: 'limel-select' },
```

Reply:
we should actually just remove this, it will automatically become `limel-select`